src/applications/piezoscan.py

```diff
@@ -339,8 +353,13 @@ def go_to_position(self, event = None):
             self.counter_scanner.stage_controller.go_to_position(x = self.view.sidepanel.go_to_x_position_text.get(), y = self.view.sidepanel.go_to_y_position_text.get())
         else:
             print(f'stage_controller would have moved to x,y = {self.view.sidepanel.go_to_x_position_text.get():.2f}, {self.view.sidepanel.go_to_y_position_text.get():.2f}')
-        self.optimized_position['x'] = self.view.sidepanel.go_to_x_position_text.get()
-        self.optimized_position['y'] = self.view.sidepanel.go_to_y_position_text.get()
+        x, y  = self.view.sidepanel.go_to_x_position_text.get(),  self.view.sidepanel.go_to_y_position_text.get()
```

Comment:
Why was this changed from __init__ to here?  It could certainly be the better place to do this, so I am asking out of curiosity.

Reply:
I did it this way since I needed the x,y position for a future use. This prevents 2 calls to the get function which sacrifices speed for memory as I store the x,y as variables.